### ISSUE TITLE
Feign Client uses java client if URL is passed

### ISSUE THREAD

**Issue by tanmaybinaykiya** [spring-cloud/spring-cloud-netflix#1255]
Whenever we pass the url in the client declaration, we see that the client uses a java client (I uses  Java/1.8.0_51 as user agent) 

On removing the url(zookeeper discovery), the client uses the okhttp3 client (user agent : okhttp3) 

`@FeignClient(name="client", url="http://localhost:8887", configuration=ClientConfiguration.class)`

`
	@Bean
	public Client feignClient(CachingSpringLoadBalancerFactory cachingFactory, SpringClientFactory clientFactory, okhttp3.OkHttpClient okHttpClient) {
		logger.info("okhttp3 passed to feign " + okHttpClient);
		OkHttpClient delegate = new OkHttpClient(okHttpClient);
		return new LoadBalancerFeignClient(delegate, cachingFactory, clientFactory);
	}`

Is this a known issue or am I missing something?

**Comment by dsyer** [comment]
It's not an issue really, it's by design. The load balanced client is different than  the "direct" one. Is that a problem for you?

**Comment by tanmaybinaykiya** [comment]
Yes, as per our use case we need to use the okhttp client without the load balanced client. Is there a way that can be achieved?
